backend/cinema_service/settings.py

```diff
@@ -175,3 +177,10 @@
     "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
     "ROTATE_REFRESH_TOKENS": False,
 }
+
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5173",
+    "http://127.0.0.1:5173",
+]
+
+CORS_ALLOW_ALL_ORIGINS = True
```

Comment:
Why define specific allowed origins only to later permit all origins?

Reply:
Forget about this
I have some issues with tasks and trying debug them